companies/ruzhang/Mario Kart 8 Deluxe.php

```diff
@@ -87,6 +87,10 @@
          </div> <!-- /entry -->
 
       </div> <!-- /portfolio-content -->
+      <?php
```

Comment:
Put this code block in the SAME POSITION PLZ. Every single one of them.

Reply:
I don't get this one.